pkg/lib/cockpit-components-form-helper.tsx

```diff
@@ -24,8 +24,8 @@ import { HelperText, HelperTextItem } from "@patternfly/react-core/dist/esm/comp
 
 export const FormHelper = ({ helperText, helperTextInvalid, variant, icon, fieldId } :
   {
-      helperText?: string,
-      helperTextInvalid?: string,
+      helperText?: string | null,
+      helperTextInvalid?: string | null,
```

Comment:
Ah, the `?` doesn't allow `null` by itself? That's rather strict. (I see it making sense both ways, TIL)

Reply:
the `?` allows `undefined` but I've written code where it's more comfortable for this to be `null`.